perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py

```diff
@@ -70,13 +72,38 @@
 
 
 # Stress test options.
-CONSISTENCY_LEVEL = 'quorum'
+CONSISTENCY_LEVEL = 'QUORUM'
 REPLICATION_FACTOR = 3
 RETRIES = 1000
 
 CASSANDRA_STRESS = posixpath.join(cassandra.CASSANDRA_DIR, 'tools', 'bin',
                                   'cassandra-stress')
 
+# Results documentation:
+# http://docs.datastax.com/en/cassandra/2.1/cassandra/tools/toolsCStressOutput_c.html
+RESULTS_METRICS = (
+    'op rate',  # Number of operations per second performed during the run.
+    'partition rate',  # Number of partition operations per second performed
+                       # during the run.
+    'row rate',  # Number of row operations per second performed during the run.
+    'latency mean',  # Average latency in milliseconds for each operation during
+                     # that run.
+    'latency median',  # Median latency in milliseconds for each operation
+                       # during that run.
+    'latency 95th percentile',  # 95% of the time the latency was less than
+                                # the number displayed in the column.
+    'latency 99th percentile',  # 99% of the time the latency was less than
+                                # the number displayed in the column.
+    'latency 99.9th percentile',  # 99.9% of the time the latency was less than
+                                  # the number displayed in the column.
+    'latency max',  # Maximum latency in milliseconds.
+    'Total partitions',  # Number of partitions.
+    'Total errors',  # Number of errors.
+    'Total operation time')  # Total operation time.
+AGGREGATED_METRICS = {'op rate', 'partition rate', 'row rate',
```

Comment:
A link to the docs is great. Could you briefly document `AGGREGATED_METRICS` and `MAXIMUM_METRICS`?